SOOP/SOOP_ASF_SST_XBT/soop_xbt_pipeline/create_plot.py

```diff
@@ -5,25 +5,27 @@
 # Institute: IMOS / eMII
 # email address: laurent.besnard@utas.edu.au
 
-import sys, os
-import tempfile
+import datetime
+import itertools
+import os
+import re
 import shutil
+import string
+import sys
+import tempfile
+
+import matplotlib
+import matplotlib.pyplot as plt
+import pylab
+from matplotlib.pyplot import figure, plot, show, title, xlabel, xticks, ylabel
+from netCDF4 import Dataset, num2date
+from numpy import ma
 
 # the next 3 lines are needed to create plots without a xserver
 MPLCONFIGDIR = tempfile.mkdtemp()
 os.environ['MPLCONFIGDIR'] = MPLCONFIGDIR
-import matplotlib
```

Comment:
This might need a bit more thought... my guess is `MPLCONFIGDIR` needs to be set before importing `matplotllib`. So either we make an exception for the import ordering here (not ideal) or we set this environment variable elsewhere before calling this script. The second option would make sense, as I don't think it 's good practice to be setting environment variables inside scripts.